### ISSUE TITLE
What is the latest version?

### ISSUE THREAD

**Issue by ricardocosta** [gcedo/eventsourcemock#5]
Hello!

I noticed NPM registry has more versions for this package than the tags available here on GitHub. Was that intended?

The result of running `npm info eventsourcemock time` is
```javascript
{
  modified: '2018-04-29T20:34:16.111Z',
  created: '2017-06-09T12:21:21.619Z',
  '1.0.0': '2017-06-09T12:21:21.619Z',
  '1.0.1': '2017-06-09T13:58:48.377Z',
  '1.1.0': '2018-01-07T18:52:56.546Z',
  '2.0.0': '2018-04-29T20:34:12.778Z'
}
```

Also, since the latest version bump was a major, is there any changelog I can check for the changes?

Thank you!

**Comment by clisterdmello** [comment]
2.0.0 seems to be the latest version.

**Comment by ricardocosta** [comment]
@clisterdmello Yes, that hasn't changed since the original issue report. The issue is that NPM says there are more versions than Github tags. And `package.json` says `1.0.1`. All this inconsistency doesn't seem "correct".
